src/interface/src/app/treatments/treatment-project-area/treatment-project-area.component.ts

```diff
@@ -34,6 +37,38 @@ export class TreatmentProjectAreaComponent implements OnDestroy {
   ) {}
 
   opacity = this.mapConfigState.treatedStandsOpacity$;
+  projectAreaId$ = this.treatmentsState.projectAreaId$;
+
+  treatedStands() {
```

Comment:
Here I'm grabbing the total treatedStands, but I suspect we might need to grab this and totalStands() from the treatmentsState in a few places. Should we just make this a convenience method in treatmentsState?

Reply:
what do you need here, just the number of treatedStards ? you should have this on TreatedStandsState no? Theres also something similar over MapProjectAreasComponent.getPrescriptionStandCount if you want the count for a specific project area.
If this is similar then I agree that would be great to not have this behavior repeated!

Reply:
ok, i'll look again at these, I could have missed something obvious

Reply:
(also, same comment here, given that we want to resolve the observable with async is best to use a property instead of a function, which will be called again each time angular detects changes)